runtime/queries/nix/indents.scm

```diff
@@ -1,18 +1,25 @@
+
 [
-  ; "function",
-  (bind)
-  (assert)
-  (with)
+  ; Bracket like
   (let)
-  (if)
-
   (attrset)
-  (list)
-  (indented_string)
+  (rec_attrset)
+  (let_attrset)
   (parenthesized)
+  (list)
+
+  ; Binding
+  (bind)
+  (inherit)
+  (inherit_from)
+  (formal)
+
+  ; Binary operations
+  (binary)
+  (has_attr)
+  (select)
+  (app)
 ] @indent
 
-[
-  "}"
-  "]"
-] @outdent
```

Comment:
We should probably keep these unless the injects work better without them

Reply:
Not sure exactly why tbh, but I was getting extra indents with the previous definitions. These ones seem to work more accurately